firstvoices/backend/search/indexing/base.py

```diff
@@ -0,0 +1,229 @@
+from logging import getLogger
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.utils import timezone
+from elasticsearch.exceptions import ConnectionError, NotFoundError
+from elasticsearch.helpers import actions
+from elasticsearch_dsl.connections import connections
+from elasticsearch_dsl.index import Index
+
+from backend.search import logging
+from backend.search.utils.object_utils import search_by_id
+from firstvoices.settings import ELASTICSEARCH_DEFAULT_CONFIG
+
+
+class IndexManager:
+    index = ""
+    document_managers = []
+
+    @classmethod
+    def rebuild(cls):
+        logger = logging.logger
```

Comment:
I like the approach here, but I'm not sure why, but this instantiates the logger again. So we see statements twice in the logs. 

One way I tried that worked locally was, 
adding `logger.handlers.clear()` right after initializing the logger in `logging.py` or after instantiating the logger here.
Or we can import logging and rename the backend.search.logging something else/import as something else, and then grab the logger again `logger = logging.getLogger(ELASTICSEARCH_LOGGER)` and the log from there.

Reply:
I poked around and I think the root issue is that the name of the elasticsearch logger was not following the package hierarchy with its naming, so it was propagating strangely to the root logger. I renamed it using the __name__ of the search logging module and the duplication went away. I can update our Sentry saved search to catch these logs

Reply:
Sounds good. Nice catch and thanks for figuring that out.

Reply:
Thanks for noticing the dupes!